sync.py

```diff
@@ -72,14 +82,35 @@ def sync_directly(unsycned_builds):
         print("Removing " + build['nvr'] + ".src.rpm...")
         os.remove(build['nvr'] + ".src.rpm")
 
+local_packages = False
 # Badly written but working python script
 if __name__ == "__main__":
     # tag = sys.argv[1]
     tag = "rhel-8.2.0-candidate"
+    ctag = "dist-c8-stream-compose"
+
     brew_proxy = brew.ClientSession("http://brewhub.engineering.redhat.com/brewhub/")
-    packages_to_track = load_package_list()
-    # build = brew_proxy.getTag()
     tagged_builds = get_tagged_builds(brew_proxy, tag)
+    if local_packages:
+        packages_to_track = load_package_list()
+    else:
+        print("Contacting mbox for:", ctag)
+        cproxy = brew.ClientSession("https://koji.mbox.centos.org/kojihub/")
```

Comment:
This yields a very long list. Right now we handle only a limited set of packages so I will set `local_packages` to `true` after merging this PR.